moto/resourcegroups/models.py

```diff
@@ -221,6 +228,14 @@ def delete(self, name):
         return group
 
 
+def get_resource_type(arn):
+    if "instance/my-ec2" in arn:
```

Comment:
This only matches a specific test case, it should regex the full ARN:
`arn:aws:ec2:<region>:<accountid>:instance/<id>`